docs/docs/guides/migrating-off-rnghenabledroot.md

```diff
@@ -0,0 +1,41 @@
+---
+id: migrating-off-rnghenabledroot
+title: Migrating off RNGHEnabledRootView
+---
+
+## Update `MainActivity.java`
+
+Update your `MainActivity.java` file (or wherever you create an instance of `ReactActivityDelegate`), so that it no longer overrides the method responsible for creating `ReactRootView` instance, or modify it so that it no longer uses `RNGestureHandlerEnabledRootView`. Do not forget to remove import for `RNGestureHandlerEnabledRootView`:
+
+```java
+package com.swmansion.gesturehandler.react.example;
+
+import com.facebook.react.ReactActivity;
+- import com.swmansion.gesturehandler.react.RNGestureHandlerEnabledRootView;
+public class MainActivity extends ReactActivity {
+
+-  @Override
+-  protected ReactActivityDelegate createReactActivityDelegate() {
+-    return new ReactActivityDelegate(this, getMainComponentName()) {
+-      @Override
+-      protected ReactRootView createRootView() {
+-       return new RNGestureHandlerEnabledRootView(MainActivity.this);
+-      }
+-    };
+-  }
+}
+```
+
+## Update your JS code
```

Comment:
Add a note that react navigation already creates root container in which case adding rootview component won't be necessary

Reply:
Added in https://github.com/software-mansion/react-native-gesture-handler/pull/1936/commits/5cb102f1b2c6a5a398ce5a79629e2831a0575a26.